packages/application-shell/src/test-utils/test-utils.js

```diff
@@ -294,7 +294,26 @@ const experimentalRenderAppWithRedux = (ui, renderOptions) => {
   });
 };
 
+// Mock utilities
+const windowMocks = {
```

Comment:
I added this mocks object to the `test-utils`, which helps setting up the mock for window objects (e.g. localStorage)

Reply:
Why have you decided against: https://www.npmjs.com/package/jest-localstorage-mock. As this "custom" `windowMocks` construct supports more mocks in the future?

Reply:
I checked that out yes but felt a "bit too complicated" for this use case.

We can use that instead if you prefer to, I don't have a strong opinion.

Reply:
Oh wait, we are already using it 🤦‍♂️